packit_service/worker/checker/bodhi.py

```diff
@@ -65,27 +82,92 @@ class IsKojiBuildCompleteAndBranchConfiguredCheckService(
     ...
 
 
-class HasAuthorWriteAccess(ActorChecker, ConfigFromEventMixin):
+class HasIssueCommenterRetriggeringPermissions(ActorChecker, ConfigFromEventMixin):
+    """To be able to retrigger a Bodhi update the issue commenter should
+    have write permission on the project.
+    """
+
     def _pre_check(self) -> bool:
-        if not self.project.has_write_access(user=self.actor):
-            logger.info(
-                f"Re-triggering Bodhi update via dist-git comment in PR#{self.data.pr_id}"
-                f" and project {self.project.repo} is not allowed for the user: {self.actor}."
+        has_write_access = self.project.has_write_access(user=self.actor)
+        if self.data.event_type in (
+            IssueCommentEvent.__name__,
+            IssueCommentGitlabEvent.__name__,
+        ):
+            logger.debug(
+                f"Re-triggering Bodhi update through comment in "
+                f"repo {self.project_url} and issue {self.data.issue_id} "
+                f"by {self.actor}."
             )
-            return False
+            if not has_write_access:
+                msg = (
+                    f"Re-triggering Bodhi update through comment in "
+                    f"repo **{self.project_url}** and issue **{self.data.issue_id}** "
+                    f"is not allowed for the user *{self.actor}* "
+                    f"which has not write permissions on the project."
+                )
+                logger.warning(msg)
```

Comment:
I would make this info, since this case can easily happen or be an accident. Please advise what was your reason to pick warning

Reply:
It is always hard to decide the level of a log 😅 I just thought that I want to read this message when info messages are turned off. I felt it to be more important than just an info. But probably you are right this could be too much noise. So let it be an info!